docs/devops.md

```diff
@@ -30,12 +45,60 @@ Our deployment pipeline is defined in the YAML file `phdi-azure/.github/workflow
 | end-to-end      | Run end-to-end tests to ensure the pipeline functions as expected within Azure.                                          | Pushes to `main`, or manually.                                                  | Verify the pipeline records the correct number of successes and failures and that data can be queried from the FHIR server. |
 | destroy         | Destroy an environment within Azure.                                                                                     | Manual                                                                          | Destroys a given Terraform environment.                                                                                     |
 
-## New Releases
-It's important to keep your repository up-to-date with version changes from the main `phdi` repository. Even if you're not using new features of the services, staying up to date is a security best practice.
 
-To update, visit the [main phdi repository](https://github.com/CDCgov/phdi) and copy the [latest version number](https://github.com/CDCgov/phdi/releases). Update the container image tag in [`main.tf`](https://github.com/CDCgov/phdi-azure/blob/main/terraform/modules/shared/main.tf#L201-L201).
+## Environments
+
+The deployment pipeline is capable of deploying the starter kit to any number of environments. This allows users to flexibly configure however many distinct instances of the starter kit they need (dev, test, staging, prod, etc.). Currently all environments are deployed within a single Azure resource group. The environment name is included in the names of individual resources to distinguish which environment they belong to. If the starter kit is initially deployed by following the [Implementation Guide](implementation-guide.md), as recommended, it will only have a `dev` environment. To create additional environments follow the steps below:
+1. Navigate to `Settings` on your version of the `phdi-azure` repository in GitHub.
+2. Click on `Environments` in the side bar.
+3. Click on `New Environment` in the top right.
+![make-new-env-1](./images/make-new-env-1.png)
+4. Enter the name of your new environment and click `Configure environment`.
+![make-new-env-2](./images/make-new-env-2.png)
+
+After following these steps to initialize a new environment see the [Deployments](#deployments) sections below for guidance on how to deploy the starter kit to it.
```

Comment:
I think this is a really good addition for this document

Reply:
Thanks!